sqlalchemy_views/views.py

```diff
@@ -83,7 +92,13 @@ class DropView(_CreateDropBase):
 
     def __init__(self, element, on=None, bind=None,
                  cascade=False, if_exists=False):
-        super(DropView, self).__init__(element, on=on, bind=bind)
+        try:
+            super(DropView, self).__init__(element, on=on, bind=bind)
+        except TypeError:
+            # Since version 1.4.0 of SQLAlchemy the ** on ** parameter no
+            # longer exists. it causes a ** TypeError ** exception
+            super(DropView, self).__init__(element, bind=bind)
```

Comment:
We should probably add a test here for `if on is not None` and raise an exception specifying that on is not supported on SQLAlchemy 1.4+. I'm concerned about silently ignoring on once folks upgrade to SQA 1.4.